src/bp/ui-shared/src/translations/index.tsx

```diff
@@ -73,7 +72,18 @@ const getUserLocale = () => {
   return translations[locale] ? locale : defaultLocale
 }
 
-const lang = (id: string, values?: Record<string, string | PrimitiveType>): string => {
+/**
+ * Can either receive an ID, or an object with keys of supported languages
+ */
+const lang = (id: string | { [lang: string]: string }, values?: { [variable: string]: any }): string => {
+  if (!id) {
+    return ''
+  }
+
+  if (typeof id === 'object') {
+    return id[locale] || id[defaultLocale] || ''
+  }
```

Comment:
Since that specific method doesn't rely on react-intl for the translation, it doesn't have the mechanism of "if lang is not set, then use the default key". It just gives an additional flexibility to translations